src/components/WelcomeHeader/WelcomeHeader.jsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react'
+import "./WelcomeHeader.scss"
+import UserIcon from '../UserIcon/UserIcon';
+
+
+const WelcomeHeader = ({ userProfile }) => {
+
+    const {firstName, lastName, userIcon } = userProfile;
```

Comment:
It will be useful to make sure first name & last name are lowercase and capitalised. Can you do this with JS?